### ISSUE TITLE
About fine-tuning

### ISSUE THREAD

**Issue by lkyee** [Paper99/SRFBN_CVPR19#6]
Hi，
From the paper,I noticed the sentence “ We also observe that ﬁne-tuning on a network pretrained on the BI degradation model leads to higher PSNR values than training from scratch. ”Could you tell me the details of "fine-tuning".
Look forward to your reply! Thanks!

**Comment by Paper99** [comment]
Hi,
We use DIV2K training dataset for fine-tuning with our proposed curriculum strategy. The pretrained network is trained on the BI degradation model. `learning_rate` is still set to 0.0001 as EDSR\RDN\RCAN do. `num_epochs` is set to 200 for a fair comparison with RDN.

**Comment by lkyee** [comment]
Thanks for your reply!
That is to say,you first use DIV2K+Flickr2K as the training set,the initial learning rate is 0.0001,and train 1000epochs,the learning rate multiply 0.5 every 200 epochs.After the train,you can get the pretrained models.And then,you use DIV2K training dataset for fine-tuning,the learning rate set to 0.0001,and train 200epochs to get the final model.
I want to know is it right?
Look forward to your reply!Thanks!

**Comment by Paper99** [comment]
Yes, it is right.
